setup.py

```diff
@@ -138,7 +147,14 @@ def get_extensions():
             include_dirs=tests_include_dirs,
             define_macros=define_macros,
             extra_compile_args=extra_compile_args,
-        )
+        ),
+        extension(
+            "torchvision._custom_ops",
```

Comment:
Humm, I'm wondering if this is the reason why the tests are failing on Windows with
```
LINK : error LNK2001: unresolved external symbol PyInit__custom_ops
build\temp.win-amd64-3.6\Release\tools\miniconda3\conda-bld\torchvision_1568036619998\work\torchvision\csrc\custom_ops\_custom_ops.cp36-win_amd64.lib : fatal error LNK1120: 1 unresolved externals
```